src/boots/boots.gd

```diff
@@ -0,0 +1,45 @@
+extends Node2D
+
+
```

Comment:
Variable, function, and signal names are too long

Reply:
Pushed a Shortening commit to keep to 3 word max, what are thoughts?

Reply:
It's good
however don't use numbers in variable names
also "note" could be replaced with "push" to more accurately describe the function